libflux/flux-core/src/semantic/nodes.rs

```diff
@@ -610,7 +610,8 @@ pub struct ImportDeclaration {
     pub loc: ast::SourceLocation,
 
     pub alias: Option<Identifier>,
-    pub path: StringLit,
+    pub registry: Option<String>,
```

Comment:
Nit: This could maybe be an enum to make it clearer

```
enum Registry {
    Stdlib,
    Other(String),
}
```

Reply:
I like this change, but just wanted to note that this isn't going to end up influencing the flatbuffers representation. I think it adds clarity to the Rust code but it doesn't really add anything to flatbuffers in my opinion and the flatbuffers representation will likely be removed if we refactor the execution to happen in Rust anyway so I don't want to spend too much time on adding an enum to flatbuffers.

Reply:
I was trying to integrate this and I think I'll just merge the current version and we can reconsider this in the future.

The reason is because I started to make the change and thought, "Well the standard library isn't really a registry. So let me use a different name." Then I started going down a rabbit hole of changing names and I'd like to get this merged to unblock other work.

Reply:
Yeah, I don't think this needs to be represented in the flatbuffer, just would be nice to be clearer on the rust side.